cuML/src/dbscan/dbscan.cu

```diff
@@ -41,15 +41,15 @@ void dbscanFit(const cumlHandle& handle, double *input, int n_rows, int n_cols,
 void dbscanFit(float *input, int n_rows, int n_cols, float eps, int min_pts,
                int *labels) {
     cumlHandle handle;
-    dbscanFitImpl(handle.getImpl(), input, n_rows, n_cols, eps, min_pts, labels, handle.getStream());
+    dbscanFitImpl(handle.getImpl(), input, n_rows, n_cols, eps, min_pts, labels, 0);
```

Comment:
If I am not missing something this is just a workaround for a bug we missed in `cumlHandle_impl`. The `cumlHandle_impl` constructor misses to initialize `_userStream` to 0. IMHO adding that is is proper solution for this.

Reply:
Sure, I'll revert this change once the bug is fixed. Right now I want to check if CI passes after this.